src/ctnetlink/mod.rs

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: MIT
+
+pub mod message;
```

Comment:
Let's not expose internal module path to public.

I prefer expose all types as `netlink-packet-netfilter::conn_track::{ContrackAttr, etc}`.